deploy/olm-catalog/awx-operator/manifests/awx.ansible.com_awxs_crd.yaml

```diff
@@ -385,15 +385,6 @@ spec:
               image:
                 description: URL of the image used for the deployed instance
                 type: string
-              postgresConfigurationSecret:
-                description: Postgres Configuration secret name of the deployed instance
-                type: string
-              broadcastWebsocketSecret:
-                description: Broadcast websocket secret name of the deployed instance
-                type: string
-              secretKeySecret:
-                description: Secret key secret name of the deployed instance
-                type: string
```

Comment:
This is getting deleted because it is now duplicated.  It was added by @Spredzy [here](https://github.com/ansible/awx-operator/pull/353/files), then added again when the bundle was generated [here](https://github.com/ansible/awx-operator/pull/364)